frontendservice/src/views/Verify.vue

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+<template>
+  <div class="verify">
+    <h1 class="header">Verification status</h1>
+    <div>
+      <span id="info" v-if="!status">Your account has not been verified.</span>
+      <span id="info" v-else>You have already verified your account!</span>
+    </div>
+    <span id="info">An email will be sent to {{email}} for verification.</span>
+    <br>
+    <a class="button is-primary" v-if="isLoggedIn && !status" @click="verify">Send a verification mail</a>
+    <br>
+    <a class="button is-danger" v-if="isLoggedIn" @click="logout">Logout</a>
+    <br>
+    <br>
+    <div v-if="isLoggedIn">
+      <router-link to="/shop-owner">Continue to Home</router-link>
```

Comment:
/shop-owner or /shopowner ?? just wondering.

Reply:
This is the ShopOwnerHome page link. It has already been merged. Hence using the same. But while implementing this page (ShopOwnerHome), if you feel like changing it, you surely can, totally your call.

Reply:
Okay cool